assets/config.php

```diff
@@ -189,7 +190,9 @@
      * You can clear cache selectively by specifying the tag.
      */
     'cache' => [
+        'manager' => CacheManager::class,
```

Comment:
Is there some benefit to using a class here instead of just a boolean `override_manager` config?

If we did go this route, I'd also inline the full class so that it's clear *which* CacheManager is being used here.

From the POV of developers disabling this to use a custom class, I think it might be a bit confusing to have to remember to go to the tenancy config file to specify which cache manager you want to use. So it might make more sense to have a boolean option here. People would set that to false and then register their own override wherever they want.

Reply:
Good point. I thought about the whole overriding thing, and I think that in the end, we only need to use the custom CacheManager when using cache tags, so we shouldn't have to override the manager in the TSP – doing so in the CacheTagsBootstrapper should do it (basically the same way [CacheTenancyBootstrapper](https://github.com/archtechx/tenancy/pull/1014/files#diff-7c9f10f1416181e2f3d8294c109feba0e9816a29fdac6974c44443004448394e) was doing it before).

So when we move the 'cache' overriding from TSP to the tag bootstrapper, the user should be able to override the cache manager:
1) whenever he wants, as long as it's before handling the prefix bootstrapper (e.g. in TSP) – if he overrides the manager after the bootstrapper runs, the new manager won't have the prefixes
2) in the CacheTagsBootstrapper by setting a static property (which would be our custom cache manager by default) – since the bootstrapper 

So I think I wouldn't use the config key in the end

Reply:
I ended up using the `override_manager` config + `$addTags` approach instead